JetNews/build.gradle

```diff
@@ -34,7 +35,7 @@ buildscript {
 }
 
 plugins {
-    id 'com.diffplug.spotless' version '6.7.0'
+    id 'com.diffplug.spotless' version '6.5.2'
```

Comment:
Is there a reason you're downgrading?

Reply:
We're upgrading spotless to 6.7.0 but leaving ktlint as it is right now, because of an issue:

https://github.com/pinterest/ktlint/issues/1195